### PR TITLE
Created fcs subworkflow

### PR DESCRIPTION
**Pull request by scorreard**
## PR checklist

Closes [nf-core/genomeassembler issue 41](https://github.com/nf-core/genomeassembler)https://github.com/nf-core/genomeassembler/issues/41

- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by scorreard** [comment]
This is my first subworkflow, and I will likely create a bunch after that, so let me know anything that can be improved so that the next ones are 👌👌

**Comment by scorreard** [comment]
Those tools don't have conda env, which explains the failed conda

**Comment by scorreard** [comment]
Docker works when running the test on GitPod, I don't really know why it fails here
image

**Comment by scorreard** [comment]
suggesting @d4straub as a reviewer as you wrote both modules

**Comment by maxulysse** [comment]
> **Answer to conda failure :** Those tools don't have conda env, which explains the failed conda

You can remove conda from the tests

**Comment by maxulysse** [comment]
@scorreard solve the merged conflicts with the ci file.
We really need to sort this file properly...

File: .github/workflows/pytest-workflow.yml [hunk 1]
```diff
@@ -146,8 +146,11 @@ jobs:
             tags: subworkflows/vcf_annotate_ensemblvep_snpeff
           - profile: "conda"
             tags: subworkflows/vcf_annotate_ensemblvep
+          - profile: "conda"
+            tags: subworkflows/fasta_clean_fcs
           - profile: "conda"
             tags: islandpath
+
```
Comment (maxulysse) [inline .github/workflows/pytest-workflow.yml:153]:
```suggestion
```

**Comment by maxulysse** [comment]
Good catch, I opened twice the same PR...
:facepalm:

**Review by maxulysse — APPROVED** [review]

**Comment by scorreard** [comment]
> Good catch, I opened twice the same PR...
> 🤦

I was refreshing the page wondering why it was not working, and then realized that... It's ok, it's Friday!

**Pull request merged**